File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int j=0, k=1;
        for(int i=0; i<nums.size(); i++) {
            if(nums[i] != nums[j]) {
                nums[j+1] = nums[i];
                j++;
                k++;
            }
        }

        return k;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        Set<Integer> st = new LinkedHashSet<>();
        for(int i : nums) {
            st.add(i);
        }

        int idx = 0;
        for(int i : st) {
            nums[idx] = i;
            idx++;
        }

        return st.size();
    }
}